### ISSUE TITLE
config, training, run successfully

### ISSUE THREAD

**Issue by eeric** [Duankaiwen/LSNet#11]
2021-05-25 06:37:44,269 - mmdet - INFO - Epoch [1][1750/29317]  lr: 1.000e-02, eta: 3 days, 4:50:47, time: 0.766, data_time: 0.005, memory: 4482, loss_cls: 0.8888, loss_bbox_init: 0.4031, loss_bbox_refine: 0.8448, loss: 2.1367, grad_norm: 1.6092
2021-05-25 06:38:22,688 - mmdet - INFO - Epoch [1][1800/29317]  lr: 1.000e-02, eta: 3 days, 4:46:34, time: 0.768, data_time: 0.005, memory: 4482, loss_cls: 0.8721, loss_bbox_init: 0.3939, loss_bbox_refine: 0.8263, loss: 2.0924, grad_norm: 1.6243
2021-05-25 06:39:01,561 - mmdet - INFO - Epoch [1][1850/29317]  lr: 1.000e-02, eta: 3 days, 4:43:59, time: 0.777, data_time: 0.005, memory: 4482, loss_cls: 0.8509, loss_bbox_init: 0.3926, loss_bbox_refine: 0.8214, loss: 2.0648, grad_norm: 1.5601
2021-05-25 06:39:40,492 - mmdet - INFO - Epoch [1][1900/29317]  lr: 1.000e-02, eta: 3 days, 4:41:41, time: 0.779, data_time: 0.005, memory: 4482, loss_cls: 0.8548, loss_bbox_init: 0.3930, loss_bbox_refine: 0.8219, loss: 2.0697, grad_norm: 1.7333
2021-05-25 06:40:19,645 - mmdet - INFO - Epoch [1][1950/29317]  lr: 1.000e-02, eta: 3 days, 4:40:07, time: 0.783, data_time: 0.005, memory: 4482, loss_cls: 0.8701, loss_bbox_init: 0.3845, loss_bbox_refine: 0.8124, loss: 2.0669, grad_norm: 1.5948
2021-05-25 06:40:58,237 - mmdet - INFO - Epoch [1][2000/29317]  lr: 1.000e-02, eta: 3 days, 4:36:58, time: 0.772, data_time: 0.005, memory: 4482, loss_cls: 0.8666, loss_bbox_init: 0.3861, loss_bbox_refine: 0.8200, loss: 2.0727, grad_norm: 1.6296

1.training script:
./code/tools/dist_train.sh ./code/configs/lsnet/lsnet_bbox_r50_fpn_1x_coco.py 2 --work-dir work_dir/lsnet_bbox_r50_fpn_1x_coco here, modified as following:
./code/configs/lsnet/lsnet_bbox_r50_fpn_1x_coco.py  ###model
./code/configs/_base_/datasets/coco_lsvr.py  ###dataset
./code/configs/_base_/schedules/schedule_1x.py  ###training
2.about detecting pretrained model, download as following:
https://github.com/HikariTJU/LD/tree/main/configs/gfl
3.but need to train instructions for various tasks yet.

**Comment by eeric** [comment]
@Duankaiwen 
2.about detecting pretrained model, download as following:
https://github.com/HikariTJU/LD/tree/main/configs/gfl

this link wasn't fit to this project.
